bin/dev.js

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env ts-node
```

Comment:
Were you wanting `ts-node` here instead of `#!/usr/bin/env -S node --loader ts-node/esm --no-warnings=ExperimentalWarning`?

Reply:
We don't actually need the `ts-node/esm` loader since this is still a CJS plugin